### ISSUE TITLE
cod.

### ISSUE THREAD

**Issue by kauecaires02** [kauecaires02/Sistema-I.A-python-para-manipula-o-de-dados#1]
import pandas as pd
from sklearn.preprocessing
import MinMaxScaler from sklearn.cluster 
import KMeans

# Carregando dados
data = pd.read_csv('dados.csv')

# Pré-processamento dos dados
scaler = MinMaxScaler()
data_scaled = scaler.fit_transform(data)

# Inicializando o modelo de agrupamento K-Means
kmeans = KMeans(n_clusters=3)

# Treinando o modelo
kmeans.fit(data_scaled)

# Adicionando a coluna de labels no dataframe
data['cluster'] = kmeans.labels_

# Visualizando os resultados
print(data.head())
